parser/expression.go

```diff
@@ -399,6 +398,8 @@ func (self *_parser) parseLeftHandSideExpression() ast.Expression {
 	for {
 		if self.token == token.PERIOD {
 			left = self.parseDotMember(left)
+		} else if self.token == token.LEFT_BRACKET {
```

Comment:
This is just a typo, token.LEFT_BRACE should have been token.LEFT_BRACKET. It was fixed in otto (https://github.com/robertkrimen/otto/commit/33ac8bd28a37753ea1e8649cadd4196be6e55e06) but sadly after I forked it...